src/v/coproc/event_listener.cc

```diff
@@ -84,13 +103,21 @@ event_listener::event_listener(ss::sharded<pacemaker>& pacemaker)
   , _dispatcher(pacemaker, _abort_source) {}
 
 ss::future<> event_listener::start() {
-    return _client.connect().then([this] {
-        (void)ss::with_gate(_gate, [this] {
-            return ss::do_until(
-              [this] { return _abort_source.abort_requested(); },
-              [this] { return do_start(); });
-        });
+    (void)ss::with_gate(_gate, [this] {
+        return ss::do_until(
+          [this] { return _abort_source.abort_requested(); },
+          [this] {
+              return do_start().handle_exception(&err_handler).finally([this] {
```

Comment:
what is the expectation if handle_exception returns an exception (ie it isn't a kafka client exception)?

Reply:
theres no need for that , true was removed